plugins/modules/nxos_route_maps.py

```diff
@@ -332,6 +332,51 @@
                   precedence:
                     description: Set precedence field.
                     type: str
+                  next_hop:
+                    description: Set next-hop IP address (for policy-based routing)
+                    type: dict
+                    suboptions:
+                      address:
+                        description: Set space-separated list of next-hop IP addresses. Address ordering is important. Also don`t use unnecessary spaces.
+                        type: str
+                      drop_on_fail:
+                        description: Drop packets instead of using default routing when the configured next hop becomes unreachable
+                        type: bool
+                      force_order:
+                        description: Enable next-hop ordering as specified in the address parameter.
+                        type: bool
+                      load_share:
+                        description: Enable traffic load balancing across a maximum of 32 next-hop addresses
+                        type: bool
+                      peer_address: BGP prefix next hop is set to the peer’s local address. If no next hop is set in the route map, the next hop is set to the one stored in the path.
```

Comment:
The reason why pre-commit is failing (and also removing some of the doc RSTs) is because of how description is added here. The description needs to assigned to the `description` key.

Reply:
done